core/store/migrate/migrations/0200_default_values_for_last_finalized_block.sql.go

```diff
@@ -0,0 +1,71 @@
+package migrations
```

Comment:
Worst case scenario (100k records to be updated), single chain migration takes up to 1s on my local machine. Node running LogPoller on X chains will take up to `X * 1s` to migrate everything